sherpa/astro/ui/utils.py

```diff
@@ -10040,7 +10040,7 @@ def _prepare_bkg_fit(self, id, otherids=()):
 
         return fit_to_ids, datasets, models
 
-    def _get_bkg_fit(self, id, otherids=(), estmethod=None, numcores=1):
+    def _get_bkg_fit(self, id, otherids=None, estmethod=None, numcores=1):
```

Comment:
If we make this change then we should also change _prepare_bkg_fit (as we call it below with otherids) and then the _prepare_fit , _get_fit, and  _get_fit_ids routines in sherpa/ui/utils.py as they end up getting called [I haven't looked below to see if they're included in this PR]. I am a little bit concerned about _get_fit_ids as it assumes otherids is iterable, which None is not.